components/Bag.jsx

```diff
@@ -3,6 +3,11 @@ import styled from "styled-components";
 import Image from "next/image";
 
 const Bag = ({ name, image, isVegan, variants }) => {
+  var numberFormat = new Intl.NumberFormat("de-DE", {
+    style: "currency",
+    currency: "EUR",
+  });
```

Comment:
You can put the definition of numberFormat outside of the component. Then it doesn't have to be recreated on every render, making it a little bit more efficient.